note/規格書/06_資料欄位對應表.md

```
@@ -132,6 +134,8 @@ updated: 2026-08-12
 | `occupation.id` | 職業代號 | string | null | 【待決策】 | ✔ |  | 【待決策】 |
 | `occupation.name` | 職業名稱 | string | null | 【待決策】 | ✔ |  | 【待決策】 |
 | `occupation.since_day` | 起始日 | int | null | 【待決策】 | ✔ |  | 【待決策】 |
+| `decision_source` | 決策來源 | enum | `local`/`cloud`/`human` | 建角時選 |  |  | 否——MVP 新增（2026-08-16），對應《12》`DecisionProvider`，供角色面板顯示 |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 7; Actual: 8; Too many cells, extra data will be missing

(MD056, table-column-count)